### ISSUE TITLE
[BUG] When i increase more than 8 plots then in same page then getting sad emoji

### ISSUE THREAD

**Issue by dhananroy** [plotly/dash#1699]
Thank you so much for helping improve the quality of Dash!

We do our best to catch bugs during the release process, but we rely on your help to find the ones that slip through.

**Describe your context**
Please provide us your environment, so we can easily reproduce the issue.

-  replace the result of `pip list | grep dash` below
```
dash                 0.42.0
dash-core-components 0.47.0
dash-html-components 0.16.0
dash-renderer        0.23.0
dash-table           3.6.0
```
-  if frontend related, tell us your Browser, Version and OS

**Describe the bug**

**Expected behavior**

**Screenshots**

If applicable, add screenshots or screen recording to help explain your problem.
![image](https://user-images.githubusercontent.com/71282375/128484786-571b40e7-57b9-4cb5-8bad-df91cd232a2c.png)

**Comment by AnnMarieW** [comment]
Hi @dhananroy  Can you please try upgrading to the current version of Dash  and see if you are still having the same issue?


dash                | 1.21.0
--|--
dash-core-components | 1.17.1
dash-html-components | 1.1.4
dash-renderer        | 1.19.1
dash-table           | 4.12.0

**Comment by jbeyer16** [comment]
Could it be related to this?

https://plotly.com/python/webgl-vs-svg/

> Context limits: browsers impose a strict limit on the number of WebGL "contexts" that any given web document can access. WebGL-powered traces in plotly can use multiple contexts in some cases but as a general rule, **it may not be possible to render more than 8 WebGL-involving figures on the same page at the same time.**

**Comment by jaugustine04** [comment]
face the similar issues 
Following dsah app is created to display 13 charts based on the dropdown down options. After choosing the options the graph become distorted.  On the left I see a sad smiley face. The underlying data is there, if I hover on the white surface I can find the invisible data points. 

Using chrome
	Line  58: dash==2.6.2
	Line  59: dash-bootstrap-components==1.2.1
	Line  60: dash-core-components==2.0.0
	Line  61: dash-html-components==2.0.0
	Line  62: dash-renderer==1.9.1
	Line  63: dash-table==5.0.0

![image](https://user-images.githubusercontent.com/7039779/197693534-9fec0995-6727-4d9a-be73-18b732047db5.png)

**Comment by alexcjohnson** [comment]
Yes I'm assuming this is about webgl -> https://github.com/plotly/dash/issues/2740
